### ISSUE TITLE
New header

### ISSUE THREAD

**Issue by Lindaleunissen** [akvo/Akvo-Theme#396]
In line with the new homepage, we also want to update the website header. 
It'll be much simplified. 

<img width="1010" alt="Header menu" src="https://user-images.githubusercontent.com/5338081/73769590-d72b3b80-4772-11ea-9773-5caddaaf825d.png">

- [ ] The logo on the right side stays as is. (with the dot colour changing on each page load)
- [ ] The main header menu can be removed. 
- [ ] On the right hand side we'll start with the languages. Not all languages will be displayed, only the active language with attached a dropdown that shows the other available languages. (Dropdown arrow in mock up is a suggestion - you know best how to handle this from a user perspective. 
- [ ] Next to the languages there'll be a hamburger menu. Through this menu people can navigate to Impact areas, stories and blogs
- [ ] Finally there will be a "Request a consultation" button. This needs to be a HubSpot generated button that links to the "Get in touch" form

**Comment by samvthom16** [comment]
@Lindaleunissen how does the menu open on click of the hamburger? Any visuals for that as well would help.

**Comment by Lindaleunissen** [comment]
@samvthom16 At the moment I'm thinking, it'll look the same as a dropdown does in the current website header, but I'm not sure what is best practice here. Open to any ideas you have.

**Comment by Lindaleunissen** [comment]
I just realised I've forgotten about the search icon. Can you also put that in the hamburger drop down?

**Comment by samvthom16** [comment]
@Lindaleunissen : if you prefer a dropdown, could you please add a visual? Another example would be something like this: https://www.evomedia.lt/en

**Comment by Lindaleunissen** [comment]
That looks really nice. Let me check with the team. I'll get back to you on this one!

**Comment by Lindaleunissen** [comment]
@samvthom16 I've shared a few options with the team, and we've decided to go with your solution for the menu. https://www.evomedia.lt/en

I've also found this: https://tamino-martinius.github.io/ui-snippets-menu-animations/
Would be lovely if you can incorporate that. 

I noticed that the current hamburger on mobile includes the language toggle. I'm wondering if we should put the languages in the new hamburger menu as well. Do you have an opinion on that one?

Thanks!

**Comment by samvthom16** [comment]
@Lindaleunissen In my opinion, having the languages dropdown within the menu would bury it quite deep.

**Comment by Lindaleunissen** [comment]
"@Lindaleunissen In my opinion, having the languages dropdown within the menu would bury it quite deep."
Thanks for your input Sam. I'm happy to go with that, so leave them in the header menu as described above.

**Comment by samvthom16** [comment]
@Lindaleunissen can you check this testing site for the header and footer: https://orbit1.sitehub.in/akvo/

I was not able to copy the content. So this will only serve as a visual reference and the links are not working as of now. Let me know if I have missed anything.

**Comment by Lindaleunissen** [comment]
@samvthom16 
Thanks for this. Is looking good! Here's my initial feedback, there might be more after Alvaro has a look at it too, but that's not until later this week. 

Hamburger menu fold out:
Works well. I like how the background fades back. Please adjust the following though:
- Make all text in the menu sentence case in stead of upper case.
- Remove a few menu items - we only need Impact areas, stories and blogs. 
- Remove the search box. Instead put a magnifying glass as the last menu item (similar to hamburger on current mobile site) The current search function works beautiful, and this new search window is a lot less elegant. 

Footer:
The 6 columns work fine, and doesn't look too crowded. A few minor adjustments:
- Increase the line height of the links to the same height we have in the current footer (I think it might be 1.5, but please check)
- The line height of links that need more than one line could still be 1 - to show the 2 lines belong together. 
- Can you check the responsiveness? If I make my window smaller it stacks as follows: first column 1 row, second column 2 rows, third column 3 rows. It should be 2 rows all along. (When I make the window even smaller everything stacks in 2 columns of 3 rows, which is perfect.)

**Comment by Lindaleunissen** [comment]
@samvthom16 

I think I've worked out some of the remaining issues. Please see the mock up below. 
- Let's put languages in the hamburger menu after all - the header menu looks more clear that way. Since we only. have 3 menu items in here, there's plenty of room. 
- Under the languages we can have a search icon. When the icon is clicked a line appears (ideally animated from left to right) where you can put your search term. That way we get rid of the box and it looks a bit more elegant.  
- Please look at the screenshot for spacing. I think it's nice to have the menu items clustered together. Then the languages and search together, and the social media right at the bottom. 

Finally, (not in the mock up) can you make the hamburger icon itself slightly larger so it stands out better?

![Screenshot mock up](https://user-images.githubusercontent.com/5338081/74850004-52731c80-5331-11ea-8b58-c522203bbbf1.jpg)

**Comment by samvthom16** [comment]
@Lindaleunissen please can you review the test site again

**Comment by Lindaleunissen** [comment]
It's looking wonderful. I only have 2 minor adjustments.
1. Can you make the search icon the same colour dark grey as the text?
2. Can you set the social media icons so they change to Akvo green on hover - like they also do in the footer?

That's it. It's looking really nice and coming together beautifully. Thanks Sam!

**Comment by samvthom16** [comment]
@Lindaleunissen I have made those changes on the test site. Please review. 

Also I would need the hubspot code for "Request a consultation" button so that I can test it out. 

Do give me a heads up before we launch the header & footer because as of now I'm maintaining these changes on a different version. And I would need time to merge them and fix if any conflicts happen.

**Comment by Lindaleunissen** [comment]
Thanks for the changes. The social media buttons seem to wobble about a bit on hover. Can you set them either so that they stay stable at the bottom as they do on the footer. OR they don't change size, only colour.

I'll send you the code for the button through the form, as github will try to interpret the code.

**Comment by samvthom16** [comment]
@Lindaleunissen have you sent the code already?

**Comment by samvthom16** [comment]
@Lindaleunissen have included the hubspot code in the header

**Comment by Lindaleunissen** [comment]
Brilliant, thank you!

**Comment by samvthom16** [comment]
@Lindaleunissen any idea when we are going to release this?

**Comment by Lindaleunissen** [comment]
Thanks for getting the page live today, @samvthom16 , we've received some nice feedback already!

Someone spotted a problem with the header menu button on firefox though. The text in the button is black, so blends in with the background. Could you have a look at it? Thank you!

![2020-03-12_1310x144_scrot](https://user-images.githubusercontent.com/5338081/76530736-15033a00-646c-11ea-8241-9f442e047b9f.png)

**Comment by psychome004** [comment]
![request](https://user-images.githubusercontent.com/44089178/76594772-01191000-6520-11ea-9922-b4ff051d6741.png)

**Comment by Lindaleunissen** [comment]
Hiya. I've checked in with the person who flagged it to me. He said: It could be something related to my browser ... I have lots of add-ons that block tracking requests ... not sure if the button linked to hubspot is somehow blocked and renders like that.

It indeed looks fine on firefox without add-ons, so I reckon we can ignore it. Thanks though!
